packages/adp-tooling/test/unit/preview/adp-preview.test.ts

```diff
@@ -161,4 +162,37 @@ describe('AdaptationProject', () => {
             expect(next).toBeCalled();
         });
     });
+    describe('addApis', () => {
+        let server!: SuperTest<Test>;
```

Comment:
Removed with c92882d.
If the same constellation was in `src` we would need it because I don't want to make null checks when using it in tests because `tsc` cannot know that `beforeAll` will always be executed before the test cases, however, since we don't call `tsc` on the test sources it seems to be no problem.
(to be honest, I am surprised that it isn't a problem in the tests - I'd still have expected a ts error when running the tests)

Reply:
Thanks